Specs/Number/English/PercentModelPercentMode.json

```diff
@@ -0,0 +1,141 @@
+[ 
+{
+  "Input": "100%",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "100%",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "100%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "twenty percent",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "twenty percent",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "20%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "one hundred percents",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "one hundred percents",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "100%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "percents of twenty",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "percents of twenty",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "20%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "one out of three",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "one out of three",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "33.3333333333333%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "one in two",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "one in two",
+      "TypeName": "percentage",
+      "Resolution": {
+        "value": "50%"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "1/4 of",
```

Comment:
Please add the cases without the "of" to make sure they are dealt with correctly.